config-coral.ini

```diff
@@ -72,7 +72,9 @@ ImageSize = 45,45,3
 ModelPath =
 
 [PostProcessor]
-MaxTrackFrame = 5
```

Comment:
Can you update the PostProcessor section in the README?

Reply:
Yes.